_includes/home/wtw-schedule.html

```diff
@@ -12,27 +10,42 @@ <h5 class="margin-top-0">Before submitting an offer you'll need to:</h5>
     <ul class="usa-list">
       <li>Register with SAM.gov</li>
       <li>Certify with Small Business Administration (SBA) for any set-asides</li>
-      <li>Understand the soliciation</li>
+      <li>Review and understand the contract schedule soliciation</li>
       <li>Complete self-led PDF trainings: Pathways to success and Readiness Assessment</li>
     </ul>
     <p><a href="javascript:void(0)" class="usa-link">Learn more about Getting on Schedule</a></p>
   </div>
 </div>
 
-<div class="grid-row grid-gap margin-top-6">
+<h4>Sell directly with Contract Schedules</h4>
+<div class="grid-row grid-gap">
   <div class="tablet:grid-col-4">
-    <div class="question-circle grid-col-1">1</div>
-    <h5 class="font-heading-md">Assemble & submit offer</h5>
-    <p>Once you're ready, you will assemble all required documents and submit your business proposal to get on GSA's schedule.</p>
+    <h5 class="font-heading-md">1. Assemble & submit offer</h5>
+    <p>
+      <ul class="usa-list">
+        <li>Find the right schedule opportunities for your business.</li>
+        <li>Identify and meet requirements</li>
+        <li> Gather and submit required documents</li>
+        <li>Develop and submit business plan and submit and offer</li>
+      </ul>
+    </p>
   </div>
   <div class="tablet:grid-col-4">
-    <div class="question-circle grid-col-1">2</div>
-    <h5 class="font-heading-md">Review & negotiate</h5>
-    <p>Your offer will be reviewed and you will be able to review and negotiate changes to the contract schedule.</p>
+    <h5 class="font-heading-md">2. Review & negotiate</h5>
+    <p>
+      <ul class="usa-list">
+        <li>Your offer will be reviewed and required documents will be reviewed.</li>
```

Comment:
[request] Previously, these bullet points didn't end in periods. Right now, the first step's bullets still do not, but now the second and third do. I don't mind either way, but could you standardize them with whichever you prefer?